sklearn/utils/_tags.py

```diff
@@ -393,7 +393,27 @@ def get_tags(estimator) -> Tags:
     tag_provider = _find_tags_provider(estimator)
 
     if tag_provider == "__sklearn_tags__":
-        tags = estimator.__sklearn_tags__()
+        try:
+            tags = estimator.__sklearn_tags__()
+        except AttributeError as exc:
+            if str(exc) == "'super' object has no attribute '__sklearn_tags__'":
```

Comment:
I think checking for the string could be fragile. Could we do this instead?

```suggestion
            if hasattr(estimator, "__sklearn_tags__"):
```

Reply:
that always returns true if inheriting from baseestimator

Reply:
But then, you should not have trigger the `AttributeError` at the first place.

Reply:
Actually @adrinjalali is right here. In the case, that you inherit from `BaseEstimator` and for some reason, an error is raised, then, this approach will not reraise the error. I did not think.